docs/tools/FastqFilter.md

```diff
@@ -0,0 +1,34 @@
+# FastqFilter
+
+## Introduction
+ 
```

Comment:
This will be fixed in the next version when this get fetched from the source directly, can be ignored for this merge